galley/docker/Dockerfile.galley

```diff
@@ -9,7 +9,7 @@ COPY galley /usr/local/bin/
 # 1001 is chosen as the UID to be added to the istio group as this is the UID that galley in the docker container runs as.
 RUN addgroup --system --gid 1001 istio && \
     adduser --system --uid 1001 --gid 1001 istio && \
-    chmod -R 760 /usr/local/bin/galley
```

Comment:
We have been down this rabbit hole already.

1. Please set permissions during the build process in the makefile
1. if that doesn't work please explain why not

We can proceed from there.

Reply:
Yes. I think it should be working during the build process. Let me try it out.